subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_aux.c

```diff
@@ -429,14 +486,18 @@ static inline int isr_rx_pdu(struct lll_adv_aux *lll_aux,
 	pdu_adv = lll_adv_data_curr_get(lll);
 	pdu_aux = lll_adv_aux_data_latest_get(lll_aux, &upd);
 
-	if (pdu_adv->type == PDU_ADV_TYPE_EXT_IND) {
-		/* AdvA is placed at 2nd byte of ext hdr data */
-		addr = &pdu_aux->adv_ext_ind.ext_hdr_adi_adv_data[1];
-		tx_addr = pdu_aux->tx_addr;
+	/* AdvA is placed at 2nd byte of ext hdr data */
```

Comment:
Yes, since this code handles aux PDUs it means we always have `ADV_EXT_IND` on primary, otherwise we have legacy advertising and no aux PDUs to handle. 

btw, this change was already merged as a part of some other PR iirc so will be removed from this PR